Cargo.toml

```diff
@@ -10,60 +10,34 @@ path = "src/lib.rs"
 crate-type = ["staticlib"]
 
 [dependencies]
-# algebra = {version = "0.1.0", git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development"}
-# primitives = {version = "0.1.0", features = ["merkle_tree"], git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development"}
-# proof-systems = {version = "0.1.0", features = ["groth16"], git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development"}
-
-# r1cs-crypto = {version = "0.1.0", git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development", optional = true}
-# r1cs-core = {version = "0.1.0", git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development", optional = true}
-# r1cs-std = {version = "0.1.0", git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "development", optional = true}
+#algebra = {version = "0.1.0", git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "algebra_ocl", rev = "942f6780b1072911dcd03eeb260cb283b7c34a8a", features = ["tweedle"]}
+#primitives = {version = "0.1.0", features = ["merkle_tree", "tweedle"], git = "https://github.com/HorizenOfficial/ginger-lib.git", branch = "algebra_ocl", rev = "942f6780b1072911dcd03eeb260cb283b7c34a8a"}
 
 maybe-uninit = "=2.0.0"
 rand = "0.7.3"
 winapi = "=0.3.6"
-lazy_static = "=1.2.0"
+lazy_static = "=1.4.0"
 libc = "=0.2.70"
 cfg-if = "= 0.1.2"
 
 [dependencies.algebra]
 git = "https://github.com/HorizenOfficial/ginger-lib.git"
-rev = "ba60ec4b2abd85e06dc2eac7c3709d05a8432307"
+branch = "algebra_ocl"
+features = ["tweedle"]
 version = "0.1.0"
 
 [dependencies.primitives]
 git = "https://github.com/HorizenOfficial/ginger-lib.git"
-rev = "ba60ec4b2abd85e06dc2eac7c3709d05a8432307"
-features = ["merkle_tree"]
+branch = "algebra_ocl"
+features = ["merkle_tree", "tweedle"]
 version = "0.1.0"
 
-[dependencies.proof-systems]
-git = "https://github.com/HorizenOfficial/ginger-lib.git"
-rev = "ba60ec4b2abd85e06dc2eac7c3709d05a8432307"
-features = ["groth16"]
+[dependencies.cctp_primitives]
```

Comment:
Link to the Git-repo is needed instead of the local dependency for ability to build project

Reply:
Oops, typo fixed